datasync/chaindatafetcher/kafka/consumer.go

```diff
@@ -198,7 +210,7 @@ func insertSegment(newSegment *Segment, buffer [][]*Segment) ([][]*Segment, erro
 		buffer = append(buffer, []*Segment{newSegment})
 	} else {
 		// the segment may be already handled.
-		logger.Warn("the message may be inserted already. drop the segment", "segment", newSegment)
+		Logger.Printf("the message may be inserted already. drop the segment [segment: %s]\n", newSegment.String())
```

Comment:
Can we set an error level? I think this message will be printed too much when multi-producers are used.

Reply:
Same as above comment. This will be printed when the session is restarted.